R/get_intercom_data.R

```diff
@@ -1,13 +1,46 @@
 #' Get data from intercom API
-#' @param app_id App ID
-#' @param key API Key
+#' @param endpoint Object to access under https://api.intercom.io/
```

Comment:
Added some examples.
I meant names of objects in intercom service but updated it to " Name of target data"